puppeteer-pdf.js

```diff
@@ -33,15 +33,14 @@ cli
   )
   .option(
     "-f, --format [format]",
-    "Paper format. If set, takes priority over width or height options. Defaults to 'Letter'.",
-    "Letter"
+    "Paper format. If set, takes priority over width or height options. Defaults to 'Letter'."
   )
```

Comment:
Puppeteer's default is "Letter", so the default still works when users don't supply a format.